router/pkg/config/testdata/config_defaults.json

```diff
@@ -125,7 +125,10 @@
     },
     "Router": {
       "MaxRequestBodyBytes": 5000000,
-      "MaxHeaderBytes": 0
+      "MaxHeaderBytes": 0,
+      "Compression": {
+        "Enabled": false
```

Comment:
Is it more likely that someone would want decompression by default?

Reply:
Decompression is now enabled by default